docker-compose.override.yaml

```diff
@@ -8,8 +8,8 @@ services:
     ports:
       - "3003:3003"
     volumes:
-      - node_modules:/home/node/node_modules
-      - .:/home/node
+      - node_modules:/usr/src/app/node_modules
+      - .:/usr/src/app

```

Comment:
will the settings from docker-compose-override be appliedduring the CI build when running test-compose?
If so, then the volume settings should move to another file as we do not want volumes to be mounted during the CI tests, mounted volumes are only useful when working locally with nodemon.

Reply:
No, the test step in azure-pipelines.yaml specifies `docker-compose.yaml` and `docker-compose.test.yaml` so the override yaml won't be used.